.github/workflows/ci.yml

```diff
@@ -189,8 +190,14 @@ jobs:
         run: |
             make reload-supabase-schemas
 
+      - name: Test supabase
+        if: matrix.run_supabase_tests
+        shell: bash
+        run: |
+          make test-supabase
+
       - name: Test
-        if: matrix.arch != 'arm64'
+        if: matrix.arch != 'arm64' && !matrix.run_supabase_tests
```

Comment:
Maybe we should have this condition somewhere shared if possible? It would be nice if we had a shared place with a comment to explain why we don't run the tests on arm64 and the supabase-specific runner.